.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
```

Comment:
Let's also bump the Ruby version used to 3.1 below.

Reply:
done